LEGO1/lego/legoomni/include/legoactor.h

```diff
@@ -52,7 +52,7 @@ class LegoActor : public LegoEntity {
 	virtual void VTable0x58(MxFloat p_unk0x70) { m_unk0x70 = p_unk0x70; } // vtable+0x58
 
 	// FUNCTION: LEGO1 0x10002cf0
-	virtual MxFloat VTable0x5c() { return m_unk0x70; } // vtable+0x5c
+	virtual MxFloat GetUnk0x70() { return m_unk0x70; } // vtable+0x5c
```

Comment:
The name should remain `VTable0x5c`. It's a different convention compared to regular functions, but for virtual functions we decided the naming pattern with the vtable offset is preferable